src/services/__tests__/TeamAssignmentService.unit.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { createRandomTeams } from "../TeamAssignmentService";
+import { Team } from "../../types/common";
+import { BallChaserQueueBuilder } from "../../../.jest/Builder";
+
+const mockBallChasers = BallChaserQueueBuilder.many(6);
+
+//Remove team values and mmr as they will be set below.
+mockBallChasers.forEach((player) => (player.team = null));
+
+describe("Team Assignment Service tests", () => {
+  it("splits teams equally in size", async () => {
+    const sortedBallChasers = await createRandomTeams(mockBallChasers);
+
+    expect(sortedBallChasers).toHaveLength(6);
+    expect(sortedBallChasers.filter((player) => player.team == Team.Orange)).toHaveLength(3);
+    expect(sortedBallChasers.filter((player) => player.team == Team.Blue)).toHaveLength(3);
+
+    sortedBallChasers.forEach((player) => {
```

Comment:
I _think_ this part is a bit redundant given the 3 assertions above. If there are 6 players, and there's 3 on each team, then every player must have a team, right?

Reply:
Fixed in next commit.